defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+# defaults file for setup_kdump
+kdump_package: kexec-tools
+kdump_conf_path: /etc/kdump.conf
+kdump_become_user: root
+
+kdump_hostname: kdump-setup-service
+kdump_core_collector_args: makedumpfile -l --message-level 1 -d 31
+kdump_dracut_mnt_pth: kdump.usersys.redhat.com:/var/www/html/vmcore /mnt/vmcore
```

Comment:
This would depend on having an NFS server running somewhere on the network.  I suspect that many use cases would not involve an NFS server.  I think we may need to point to a local path for this, or even omit it, depending on how this option behaves by default.  We'll have to learn more about how kdump works to determine the most sensible default here.